inject.js

```diff
@@ -47,6 +47,22 @@ ytd-playlist-video-renderer:not(:hover) ytd-thumbnail,
 const elem = document.createElement("style");
 document.documentElement.appendChild(elem);
 
+const IsItWorkTime = (startTime, endTime)=>{
```

Comment:
nit: use camel case for JS functions. Align naming with options if we're renaming to `scheduledBlocking`.